utils/envvars/parser.go

```diff
@@ -135,6 +142,33 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SAAS host env variable value.
+// if it's valid, otherwise returns defaultSAASHost.
+func GetSAASHost(env string) string {
+	if v, b := os.LookupEnv(envSAASHost); b {
```

Comment:
In go it's common to name bool flags representing success of getting some value from some container `ok`.

So `b` -> `ok` will follow gostyle)

Reply:
oh, forgot about that one, thanks. Done